cart/application/cartService.go

```diff
@@ -728,8 +728,7 @@ func (cs *CartService) DeleteCartInCache(ctx context.Context, session *web.Sessi
 }
 
 // ReserveOrderIDAndSave reserves order id by using the PlaceOrder behaviour, sets and saves it on the cart.
-// If the cart already holds a reserved order id no set/save is performed and the existing cart is returned.
-// You may want to use this before proceeding with payment to ensure having a useful reference in the payment processing
```

Comment:
This last hint should stay.

Reply:
You're right - we should not remove that comment.